tools/c7n_azure/tests/test_tags.py

```diff
@@ -24,7 +24,7 @@
 class TagsTest(BaseTest):
 
     # latest VCR recording date that tag tests
```

Comment:
Can you add more details in the comments, so we can remember to change this value when we have to regenerate the cassettes?